### PR TITLE
[Coin Fmk] Currencies export cleaning

### PR DESCRIPTION
**Pull request by sprohaszka-ledger**
### ✅ Checklist

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Simplify and clarify what `currencies` part is in coin-framework and LLC.
Using more barrel import to better control what is exported from `coin-framework` to outside world.

### ❓ Context

- **JIRA or GitHub link**: N/A

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by Justkant — APPROVED** [review]
LGTM for the wallet-api changes as it only changes the import and I suppose keeps the logic the same

File: libs/coin-framework/src/currencies/chopCurrencyUnitDecimals.ts
Comment (lambertkevin) [inline libs/coin-framework/src/currencies/chopCurrencyUnitDecimals.ts:1]: Nice catch 👀 🧹

File: libs/coin-framework/src/currencies/support.ts
Comment (lambertkevin) [inline libs/coin-framework/src/currencies/support.ts:1]: Nice catch x2 👀 🧹 🕵️‍♂️

**Review by lambertkevin — APPROVED** [review]
Thanks for the clean up and the detective work 🚀 🕵️‍♂️

**Pull request merged**